styleguide/examples/PledgeCollectiveCard.md

```diff
@@ -0,0 +1,14 @@
+```js
+<PledgeCollectiveCard
+  collective={{
+  description = "Pledged Collective",
```

Comment:
This syntax is not valid https://deploy-preview-1639--oc-styleguide.netlify.com/#!/PledgeCollectiveCard